src/Hooks/GutenbergCleanup.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace WPML\PB\Elementor\Hooks;
+
+use WPML\FP\Fns;
+use WPML\FP\Maybe;
+use WPML\FP\Obj;
+use WPML\FP\Relation;
+use WPML\PB\Elementor\DataConvert;
+use WPML_Elementor_Data_Settings;
+use function WPML\FP\partial;
+use function WPML\FP\pipe;
+
+class GutenbergCleanup implements \IWPML_Backend_Action, \IWPML_Frontend_Action, \IWPML_DIC_Action {
```

Comment:
@strategio why do we need `IWPML_DIC_Action` if there is no constructor?

Reply:
Yes, this is definitely optional @dgwatkins, but it should not hurt :)

Reply:
ok 🤷

Reply:
I agree with @dgwatkins here. It doesn't hurt but adds confusion

Reply:
Removed!